payment_cycle/schema.py

```diff
@@ -0,0 +1,42 @@
+import graphene
+import graphene_django_optimizer as gql_optimizer
+from django.contrib.auth.models import AnonymousUser
+from django.db.models import Q
+
+from core.schema import OrderedDjangoFilterConnectionField
+from core.utils import append_validity_filter
+from payment_cycle.apps import PaymentCycleConfig
+from payment_cycle.gql_mutations import ProcessBenefitPlanPaymentCycleMutation
+from payment_cycle.gql_queries import PaymentCycleGQLType
+from payment_cycle.models import PaymentCycle
+
+
+class Query(graphene.ObjectType):
+    payment_cycle = OrderedDjangoFilterConnectionField(
+        PaymentCycleGQLType,
+        orderBy=graphene.List(of_type=graphene.String),
+        dateValidFrom__Gte=graphene.DateTime(),
+        dateValidTo__Lte=graphene.DateTime(),
```

Comment:
dateValidFrom and DateValidTo are not used to filter in resolve method

Reply:
`append_validity_filter` applies `dateValidFrom__Gte` and `dateValidTo__Lte` filters